LorafyAPI/Controllers/EndDeviceController.cs

```diff
@@ -1,20 +1,24 @@
-﻿using LorafyAPI.Models;
+﻿using LazyCache;
```

Comment:
Why have you imported caching?

Reply:
> Why have you imported caching?
Because I was trying to figure out how it works.